context.go

```diff
@@ -79,9 +79,7 @@ func (c *Context) R(obj interface{}) Response {
 
 // Respond Respond with an object
 func (c *Context) Respond(obj interface{}) Response {
-	c.Response.StatusCode = 200
-	c.Response.Data = obj
-	return c.Response
+	return c.Status(-1, obj)
```

Comment:
My changes to Respond and adding the `SetDefaults` function remind me a lot of my Java days. Is there a better way to accomplish this (not sure of the overhead in Go of misdirection with regards to functions calls)?